apps/etterlatte-saksbehandling-ui/client/src/components/behandling/trygdetid/detaljer/BeregnetSamletTrygdetid.tsx

```diff
@@ -19,16 +19,27 @@ export const BeregnetSamletTrygdetid = ({ beregnetTrygdetid }: Props) => (
       <Table.Body>
         <Table.Row>
           <Table.DataCell>Samlet trygdetid for nasjonal beregning</Table.DataCell>
-          <Table.DataCell>{beregnetTrygdetid.samletTrygdetidNorge}</Table.DataCell>
-          <Table.DataCell />
+          <Table.DataCell colSpan={2}>
+            {beregnetTrygdetid.samletTrygdetidNorge}
+            {beregnetTrygdetid.yrkesskade && (
+              <>
+                {' '}
+                (justert for yrkesskade
+                {beregnetTrygdetid.beregnetSamletTrygdetidNorge && (
+                  <> - opprinnelig beregnet verdi {beregnetTrygdetid.beregnetSamletTrygdetidNorge}</>
```

Comment:
Her kan man også bare returnere en string :D

Reply:
Må force ledende mellomrom også - og det vil da kreve {' '} - og da må vi uansett ha <> rundt for å ha kun en?